### ISSUE TITLE
Does Sonar 8.0 fully replace this plugin?

### ISSUE THREAD

**Issue by skestle** [AmadeusITGroup/sonar-stash#207]
Does the functionality described at https://docs.sonarqube.org/8.0/analysis/pull-request/ replace sonar-stash? Is there functionality missing?

It'd be nice to update the readme to indicate that while Sonar 7.7 breaks this plugin, 8.0 replaces it.

**Comment by t-8ch** [comment]
I didn't test 8.0 so far and don't have the time for it right now.
At a first glance it looks like it provides the same functionality.
If test it, could you give feedback here, so I can update the docs?

**Comment by 4n4n4s** [comment]
@skestle yes this would replace the plugin and also fix some issues as it's analyzing and creating a report for each branch.

For sonarqube lts (7.9) and 8.0+ you could check out my comment [here](https://github.com/AmadeusITGroup/sonar-stash/issues/202#issuecomment-573121401)
